### ISSUE TITLE
Opposite of "Go to Defintion"

### ISSUE THREAD

**Issue by fgotzens** [spyder-ide/spyder#4184]
## Description of your problem
In Microsoft Visual Studio there is a highly useful feature called ["Find all references"](https://blogs.msdn.microsoft.com/vcblog/2009/11/17/improvements-to-find-all-references-in-visual-studio-2010/) (which works basically like the opposite of the "Go To Definition" feature), which I miss in Spyder a lot. It just shows every single line throughout all the files of a project, where a definition of something (variable, constant, function, etc.) is being used.

It has already been mentioned [here](https://groups.google.com/forum/#!topic/spyderlib/qyUyzLNZXsw), but looks like no issue has been created for that so far.

**Comment by ccordoba12** [comment]
Good idea! We're planning to improve our projects support in Spyder 3.3 (to be released by the end of the summer) and this would be a great addition to that effort :-)

**Comment by ccordoba12** [comment]
@andfoy, I think we could use the improvements you've been working on for *Find in files* to implement this.

So the idea would to add context-menu action for this, then use Jedi or a regexp to find all use of `foo(bar)` for methods, or `Foo(bar)` for classes, and finally show the result in *Find in files*.

**Comment by ccordoba12** [comment]
And the idea is to make this work for the current project or directory, depending on if a project is active or not, respectively :-)

**Comment by goanpeca** [comment]
@ccordoba12 I am 👎 cause this would be adding a python specific behavior to a plugin that (maybe the only one!) that is python agnostic

**Comment by ccordoba12** [comment]
The idea is to add this to the Editor, not to *Find in files*. *Find in files* will be used only to display the result.

**Comment by goanpeca** [comment]
ok ;-)

**Comment by ianbstewart** [comment]
Any update on this? Would be great to have a "find all references" feature to help with refactoring.

**Comment by PhilipMay** [comment]
I am also interested in this feature.

**Comment by goanpeca** [comment]
@andfoy any updates/ideas on this work :-) ?

**Comment by ccordoba12** [comment]
This is work for Spyder 5.

**Comment by paraschakis** [comment]
This feature is indeed a must-have for any serious IDE.

**Comment by BLGEric** [comment]
Hope this feature can be release soon : )
Then I don't have to use other IDE just for code searching
Thanks~~

**Comment by drhowey** [comment]
Any news on this? Thanks!

**Comment by ccordoba12** [comment]
Sorry, we haven't had time to implement this but we'll try to do it next year.
